plugins/network-elements/nuage-vsp/src/com/cloud/network/element/NuageVspElement.java

```diff
@@ -370,7 +485,7 @@ public boolean applyStaticNats(Network config, List<? extends StaticNat> rules)
         try {
             try {
```

Comment:
Why is there is a nested ``try`` here?  Generally, a nested ``try`` is an antipattern.